docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '1'
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+  web:
+    build: .
+    command: bundle exec puma -C config/puma.rb
+    volumes:
+      - .:/orientation
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
```

Comment:
Does this docker-compose file work for you? I got complaints that the syntax isn't valid for version 1, but it works for version 3.

Also, if you set the ports for postgres to `"5432:5432"` then you won't have to do any fancy mapping and can revel in the glory of localhost.

Reply:
What fancy mapping? 

And do you mean setting the version to 3 works while 1 doesn't? Seems weird.

Reply:
It's the same mapping that you're using for `web`. 
```
ports:
  - "5432:5432"
```

Saw your note about the version, too 👍